config/locales/es.yml

```diff
@@ -952,7 +952,7 @@ i18n.language: Idioma
 i18n.locale.en: English
 i18n.locale.es: Español
 i18n.locale.fr: Français
-i18n.locale.zh: Chinese
```

Comment:
There's an exception for this in `i18n_spec.rb`. Should it be removed?

Reply:
https://github.com/18F/identity-idp/blob/20f6eaf493eb16da7bc1e8cdc8189f4d79fe6d44/spec/i18n_spec.rb#L79-L80

Reply:
Yep, thanks, moved it up with the other similar keys now that it's been translated.